pkg/machines/components/desktopConsole.jsx

```diff
@@ -0,0 +1,219 @@
+/*jshint esversion: 6 */
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+import React from "react";
+import cockpit from 'cockpit';
+
+const _ = cockpit.gettext;
+
+const MoreInformationInstallVariant = ({ os, command, innerHtml }) => {
```

Comment:
As I'm learning react still and am interested: Is this "declare a const lambda expression" style in any way better or more performant than the standard and [documented](https://facebook.github.io/react/tutorial/tutorial.html#functional-components) functional component syntax `function MoreInformationInstallVariant(os, command, innerHtml)`?